### PR TITLE
Add R scripts

### PR DESCRIPTION
**Pull request by johpiip**
The `strafica` utility package is still private but HELMET4 team has
a right to use it during this project. The package will be made public
once some small details have been solved. By Strafica's permission,
these scripts that use the package are now available. Ask more from me!

### CODE REVIEW

File: estimation/alternatives.R [hunk 1]
```diff
@@ -0,0 +1,223 @@
+# -*- coding: windows-1252-dos -*-
```
Comment (paasovaara) [inline estimation/alternatives.R:1]: interesting encoding . Perhaps utf-8 would be better..?

File: input/zones.R [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: windows-1252-dos -*-
+library(strafica)
+
+aluejako = read.shape(ancfile("data/raw/sijoittelualueet2019/sijoittelualueet2019.shp"),
```
Comment (paasovaara) [inline input/zones.R]: Where does this data file come from..? Perhaps add it to documentation:
- what data is expected to be located where
- OR what are the steps to produce the data

File: input/zones.R [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+                      encoding="utf-8")
+aluejako = sp.to.polys(aluejako, data.only=TRUE)
```
Comment (paasovaara) [inline input/zones.R]: I'd recommend using english language in all scripts

**Review by paasovaara — CHANGES_REQUESTED** [review]
Few things that would be good to fix:
- preferably always use only english language
- Would like to see some instructions (add to Readme.md?):
  - steps to setup the stack (R-version, etc), strafica library, any data needed?
  - Which scripts / batch files should be ran and in which order?
  - What are the outputs of each batch run

I could also test to run this with the strafica library, where could I find it..?

File: input/zones.R (hunk 2 above)
Comment (johpiip) [reply input/zones.R]: https://github.com/HSLdevcom/helmet-data-preprocessing/pull/9/commits/eb0f4a15e312d75718636714d0a4da70bd617c87 Not sure if I caught all steps but it's a start!

File: estimation/alternatives.R (hunk 1 above)
Comment (johpiip) [reply estimation/alternatives.R:1]: The `strafica` package might be encoded as UTF-8 already, so we could test switching the encoding in the scripts. However, most of our company's environment is in `windows-1252`. :( It's safer to just select Windows-1252 from RStudio's settings.

Comment (johpiip) [reply estimation/alternatives.R:1]: Oh, and these encoding comments do not actually do anything when they are run! They are just a reminder.

File: input/zones.R (hunk 3 above)
Comment (johpiip) [reply input/zones.R]: https://github.com/HSLdevcom/helmet-data-preprocessing/pull/9/commits/97742c1b113013f4e65234f709a5e2bca3ea542f

**Review by paasovaara — APPROVED** [review]

**Pull request merged**